lerna.json

```
@@ -1,4 +1,4 @@
 {
-    "packages": ["nodecg-io-*", "nodecg-io-core/dashboard", "samples/*"],
+    "packages": ["nodecg-io-*", "nodecg-io-core/dashboard", "samples/*", "@types/*"],
```

This can now be removed as the types are now in the package.
```suggestion
    "packages": ["nodecg-io-*", "nodecg-io-core/dashboard", "samples/*"],
```